tests/Application/config/packages/messenger.yaml

```diff
@@ -1,3 +1,5 @@
 framework:
     messenger:
-        default_bus: sylius_invoicing_plugin.command_bus
+        buses:
+            sylius.event_bus:
+                default_middleware: allow_no_handlers
```

Comment:
This bus should probably be defined in `src/Resources/config/config.yml` or mentioned in UPGRADE file because it will require changes in the end application